### DISCUSSION TITLE
(SOLVED BY MYSELF) Windows compile closes immediately when opened with no logs

### DISCUSSION THREAD

**Discussion by CUBEkid04** [FunkinCrew/Funkin#1926] [Compiling help]
So, I'm getting frustrated with this.
I had just recently set up my new computer and reinstalled everything needed to build the source code, the problem now is that it builds the game properly, but it immediately closes when its opened with no logs outputted. It works just fine when I build it to HTML5 however.

[VIDEO OF PROBLEM](https://www.youtube.com/watch?v=os8jF_lSVUw)

My version of Haxe on my old computer worked when I replaced the current version on my new computer

**Comment by Tr1NgleDev** [comment]
how did you fix this. i have the same problem

**Reply by RevixDev** [reply]
I think he needed to install newgrounds package by doing

`
haxelib install newgrounds
`

In the cmd prompt
